frontend/README.md

```diff
@@ -20,16 +20,20 @@ It correctly bundles React in production mode and optimizes the build for the be
 The build is minified and the filenames include the hashes.\
 Your app is ready to be deployed!
 
+## Environment Variables
+You can set the environment variables in `frontend/.env` to configure the frontend. The following variables are available:
+```javascript
+VITE_BACKEND_HOST="127.0.0.1:3000" // The host of the backend
+VITE_USE_TLS="false"               // Whether to use TLS for the backend(includes HTTPS and WSS) 
+VITE_INSECURE_SKIP_VERIFY="false"  // Whether to skip verifying the backend's certificate. Don't use this in production!
```

Comment:
dumb question: if `VITE_USE_TLS` is false, can I still do `VITE_INSECURE_SKIP_VERIFY="false"` in production? I thought the latter config will make no difference? If that's the case, we should add "Don't do this in production" for `VITE_USE_TLS` as well?

Reply:
yep, it doesn't matter what `VITE_INSECURE_SKIP_VERIFY` is set to if `VITE_USE_TLS` is false. it only makes sense when `VITE_USE_TLS` is true.